backend/app/api/ws/chat_namespace.py

```
@@ -146,6 +151,87 @@ async def _handle_token_expired(self, sid: str) -> dict:
+        payload = data
+        if not await can_access_task(user_id, payload.task_id):
+            return ChatGuideAck(error="Task not found or access denied").model_dump()
+
+        db = SessionLocal()
+        try:
+            team = (
+                db.query(Kind)
+                .filter(
+                    Kind.id == payload.team_id,
+                    Kind.kind == "Team",
+                    Kind.is_active == True,
```

Ruff (0.15.12)

[error] Avoid equality comparisons to `True`; use `Kind.is_active:` for truth checks

Replace with `Kind.is_active`

(E712)